models/experimental/functional_bert/README.md

```diff
@@ -0,0 +1,17 @@
+## functional_bert Demo
+## How to Run
+
+Use `pytest --disable-warnings models/experimental/functional_bert/demo/demo.py::test_demo[models.experimental.functional_bert.tt.ttnn_functional_bert-384-8-phiyodr/bert-large-finetuned-squad2-models/experimental/functional_bert/demo/input_data.json]` to run the demo.
```

Comment:
Can we change how the test is invoked to only expose parameters that users are allowed to change.
Here we want the demo to run with 384 seq size and 8 batch size in all cases for bert large, so this should not be part of test parameters.
The only parameter here should be path to input data.